### PR TITLE
Stats: Add an option to contact support for a classification update

### PR DESCRIPTION
**Pull request by grzegorz-cp**
Related to #80259

## Proposed Changes

* Add a mechanism to contact support and request an update of a site classification

## Testing Instructions

* due to missing site classification this can only be tested locally
* force your environment to classify sites as commercial (`isCommercial` variable in the main purchase component)
* go to stats and apply `stats/type-detection` feature flag
* trigger commercial purchase flow
* verify that the section at the bottom is visible like on the screenshot below
* remove the forced classification
* verify that the section at the bottom **is not** visible like on the screenshot below

SCR-20230906-khfp

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

**Comment by kangzj** [comment]
> due to missing site classification this can only be tested locally

We have the stickers! You could run the following commands in `wpsh` on your sandbox to override the status:

```PHP
switch_to_blog($blog_id);
add_blog_sticker('jetpack-site-is-commercial-override', $blog_id);
```

To remove the sticker:

```PHP
switch_to_blog($blog_id);
remove_blog_sticker('jetpack-site-is-commercial-override', $blog_id);
```

Likewise, we could use `'jetpack-site-is-not-commercial-override'` to force `isCommercial` to false.

File: client/my-sites/stats/stats-purchase/stats-purchase-single-item.tsx [hunk 1]
```diff
@@ -74,12 +79,31 @@ const StatsCommercialPurchase = ( {
...
 	const ButtonComponent = isWPCOMSite ? CalypsoButton : Button;
+	const [ isAdsChecked, setAdsChecked ] = useState( false );
+	const [ isSellingChecked, setSellingChecked ] = useState( false );
+	const [ isBusinessChecked, setBusinessChecked ] = useState( false );
+
+	const isOdysseyStats = config.isEnabled( 'is_running_in_jetpack_site' );
+
+	const handleClick = ( event: React.MouseEvent, isOdysseyStats: boolean ) => {
+		// TODO: replace with a support ticket
+		const emailHref = 'mailto:user@example.com?subject=Subject&body=message%20goes%20here';
```
Comment (kangzj) [inline client/my-sites/stats/stats-purchase/stats-purchase-single-item.tsx]: I had a second thought, maybe we could link to Jetpack support page ( https://jetpackme.wordpress.com/contact-support/?rel=support ) for now, which is not the best but at least works. We'll see if we could submit a ticket automatically. What do you think?

**Review by kangzj — APPROVED** [review]
Thanks for fixing another major part of the page!

The page is a bit jumpy when the panel is opened. It'll be great if we could take a look later.

https://github.com/Automattic/wp-calypso/assets/1425433/0011bde8-9316-4daf-9179-eb2594fca44d

File: client/my-sites/stats/stats-purchase/stats-purchase-single-item.tsx (hunk 1 above)
Comment (grzegorz-cp) [reply client/my-sites/stats/stats-purchase/stats-purchase-single-item.tsx]: Good idea. I'll change it and limit the width to avoid content jumping.

**Review by kangzj — APPROVED** [review]
:shipit:

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/8870167

Thank you @grzegorz-cp for including a screenshot in the description! This is really helpful for our translators.

**Pull request merged**